libc/startup/gpu/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ function(add_startup_object name)
       RUNTIME_OUTPUT_DIRECTORY ${LIBC_LIBRARY_DIR}
       RUNTIME_OUTPUT_NAME ${name}.o)
     target_link_options(${fq_target_name}.exe PRIVATE
```

Comment:
Why was GPU startup object named with `.exe` suffix

Reply:
It's just a dummy target, needed to be unique and since it's an executable, it's `.exe`.